lnwallet/channel.go

```diff
@@ -1893,6 +1893,33 @@ func (lc *LightningChannel) restorePendingRemoteUpdates(
 				payDesc.LogIndex)
 		}
 
+		var height uint64
+		var heightSet bool
+		if payDesc.EntryType != Add {
+			// Check whether this update is included on the pending commitment
+			// If so, we should set the removeCommitHeightRemote.
+			pendingRemoteCommitDiff, err := lc.channelState.RemoteCommitChainTip()
```

Comment:
Rather than looking it up again here, we can just pass it in from the caller's context, since we already fetch it to restore the pending commitment if it exists.

Reply:
Yes agree